tests/test_progress_parameter.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-

```

Comment:
Please change the unit test scripts name to `test_log_file.py`

Reply:
Done. It's a better naming. Thank.